docs/deployment.md

```diff
@@ -33,6 +33,33 @@ Consult the documentation on each service for more information on how they can b
 
 * Audit (@libero/audit)
 
+### Journal login
+
+TODO: move this somewhere else
+
+For journal authentication, a configuration setting needs to be set to ensure the redirect returns the user to the correct url after authentication. The following
+describe how to change this setting in each environment.
```

Comment:
state that `builder-configuration` changes require admin approval (means @diversemix)